timescaledb/how-to-guides/uninstall/uninstall-timescaledb.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Uninstall TimescaleDB
+excerpt: Uninstall TimescaleDB on macOS
```

Comment:
```suggestion
excerpt: Uninstall TimescaleDB
```

We'll presumably want to add the other OSs in here, too, so let's keep the header generic.